backend/db/models.py

```diff
@@ -11,6 +12,19 @@ class Event(db.Model):
     num_judges = db.Column(db.Integer)
     event_date = db.Column(db.Date)
 
+    def to_dict(self):
```

Comment:
Can we create a base class which has this method and have all the models inherit?